doc/HACKING/ReleaseSeriesLifecycle.md

```diff
@@ -48,3 +50,49 @@ deprecations of individual versions.
    number from their approved versions list.
 
 7. Update the CoreTorReleases wiki page.
+
+8. Update the git/hooks
+
+9. Finally, make sure this document is up to date with our latest
+   process.
+
+10. Open a ticket (if there is not one already) for authorities to
+    start rejecting relays that are running that release series.
+    This ticket should be targeted for at least a month or two
+    after the series is officially EOL, unless there is an important
+    reason to un-list relays early.
+
+11. Open a ticket (if appropriate) for updates to the set of
+    required and recommended subprotocol versions.
+
+Starting a new release series.
+------------------------------
+
+(Ideally, do this immediately after a release.)
+
+1. Start a new maint-x.y.z branch based on master, and a new
+   release-x.y.z branch based on master. They should have the same
+   starting point.
+
+   Push both of these branches to the master git repository.
+
+2. In master, change the version to "0.x.y.0-alpha-dev". Run the
+   update_versions.py script, and commit this version bump.
+
+3. Tag the version bump with "tor-0.x.y.0-alpha-dev". Push the tag
+   and master.
+
+4. Open tickets for connecting the new branches to various other
+   places, including:
```

Comment:
Let's just say 'see the list under "Open tickets to remove the release from:" above' ?

Reply:
There isn't much point in having a duplicate list, they'll just get out of sync.